### PR TITLE
Style

### PR DESCRIPTION
**Pull request by Chike1990**
We added style to the Awesome Books app, which is a basic website that allows users to add/remove books from a list. This is achieved using plain HTML, JavaScript objects, arrays, DOM, and basic events.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+
+<body>
+    <div class="container mt-4">
+        <h1>All Awesome Books</h1>
+        <table class="table table-striped mt-5">
+
+            <tbody id="book-list"></tbody>
+
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please remove the extra blank lines and indentations in the file to maintain your code and the website optimized.
_Kindly keep in mind that we're only adding them for good reasons such as separating two logical pieces of code or two large sections apart. 🙏 🥰_
_Please apply this to all similar situations through the file._

>You can find about this and more at this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct. 💯 
- Descriptive README file and PR summary. 💯 
- Good follow to the template layout. 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html]: nicely done 👍

**Comment by Davidosky007** [comment]
Hi, @Chike1990, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**